src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -586,26 +489,51 @@ public void flush() {
 		this.cubeIO.flush();
 	}
 
-	public void unloadCube(Cube cube) {
-		// don't unload cubes near the spawn
-		if (cubeIsNearSpawn(cube)) {
-			return;
+	void takeOutGarbage(){
+
+		Iterator<Cube> cubeIt = cubeMap.iterator();
+		while(cubeIt.hasNext()) {
```

Comment:
Unloading all cubes that can be unloaded every tick is a really bad idea for performance. Unloading can be quite slow. That's why it's queued.